vuepress/docs/next/tutorials/create/strapi-content.md

```diff
@@ -0,0 +1,83 @@
+# Strapi Content Management
+
+## Overview
+
+This tutorial demonstrates how to create and manage the content in your Strapi instance from the App Builder user interface.
+
+The Strapi Content Widget and Strapi Content List Widget enable the user to add or edit content and content lists, respectively, where it is mandatory to assign templates to selected content. The Strapi Content List Widget also allows content to be saved as a search. The following table compares widget capabilities.
+
+| Strapi Content Widget |  Strapi Content List Widget
+|:--|:--
+| User can select only one content | User can select more than one content |
+| User can select only one template per selected content | User can select different templates for different content |
+| There is no `Save as Query` feature | The `Save as Query` feature allows the user to save multiple contents |
+
+## Prerequisites
+
+- [A working instance of Entando](../../../docs/getting-started/)
+- Verify dependencies with the [Entando CLI](../../docs/getting-started/entando-cli.md#check-the-environment): `ent check-env develop`
+- [A configured Strapi instance running on Entando](../../tutorials/solution/strapi.md)
+
+## Strapi Content Widget
+
+1. Select a page or [create a page](../compose/page-management.md#create-a-page) in your Entando Application on which to place the Strapi Content Widget
+2. Go to  `App Builder` → `Pages` → `Management`
+3. Find the Strapi content page in the page tree and click on the three dots representing the `Actions` icon
+4. Select `Design` from the drop-down
+5. Click on the `Widgets` tab in the right panel and expand the `User` section
+6. Drag and drop the Strapi Content Widget into an empty frame in the Page Designer to load the configuration page
+7. From the configuration page that loads from the Page Designer, click `Add existing content`
```

Comment:
How do they get to the configuration page? What does it mean "that loads from the Page Designer" Is it automatic?

Reply:
yes! automatic. it felt unnecessary to add 'automatic' to that sentence. nothing is required of the user so it will make sense as they progress thru the tutorial - no clarification needed: clear in retrospect and retrospect is guaranteed